### DISCUSSION TITLE
how do I find all human "positive regulator of" annotations in one text file?

### DISCUSSION THREAD

**Discussion by a-dna-n** [geneontology/helpdesk#442] [General]
Hi.

I'd like to download a file from which I can extract all human genes with "positive regulator of" annotations, like [GO:0060545](https://www.ebi.ac.uk/QuickGO/annotations?goUsage=descendants&goUsageRelationships=is_a,part_of,occurs_in&goId=GO:0060545), or construct those links if need be. 

I'm not looking for an API, because I don't know the terms I'm looking for in advance. I've tried gpad and gaf files to no avail. The answer I found on my own is "Panther", so I must be looking in the wrong places. Thanks!

**Comment by balhoff** [comment]
@bzip2 I think the AmiGO site might be of use for you: http://amigo.geneontology.org/amigo/term/GO:0048518

That page is for [positive regulation of biological process](http://purl.obolibrary.org/obo/GO_0048518), and shows all annotations for that term as well as all its subclasses (and parts, in the default configuration). You can change the filters on the left side (e.g., select Human), and then download all the results.
